src/main/kotlin/com/openlattice/datastore/services/EntitySetService.kt

```diff
@@ -99,7 +99,7 @@ open class EntitySetService(
 
 
     override fun createEntitySet(principal: Principal, entitySet: EntitySet) {
-        val entityType = entityTypes.getValue(entitySet.entityTypeId)
+        val entityType = Util.getSafely(entityTypes, entitySet.entityTypeId)
```

Comment:
Why the switch? The left over Util.getSafely in this class is from the original java implementation. Kotlin .getValue is typesafe without needing utility method.

Reply:
Because the implementation of `.getValue()` on an `IMap` also calls `.containsKey`
```
internal inline fun <K, V> Map<K, V>.getOrElseNullable(key: K, defaultValue: () -> V): V {
    val value = get(key)
    if (value == null && !containsKey(key)) {
        return defaultValue()
    } else {
        @Suppress("UNCHECKED_CAST")
        return value as V
    }
}
```

Reply:
As a follow-up, I used `getSafely` specifically because there was a pattern of using it in `EntitySetService` I'm happy to deprecate that method and just use `.get` if you think that's the way to go moving forward